src/index.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<html>
+<head>
+    <title>Idea Town</title>
+    <!-- inject:css -->
```

Comment:
Yeah, [gulp-inject](https://www.npmjs.com/package/gulp-inject) is at work here. Seemed like the simplest way to get started, after I decided to abandon the magic and mystery of webpack and its webpack-dev-server